src/PYTHON/fix_python_invoke.cpp

```diff
@@ -92,8 +92,9 @@ int FixPythonInvoke::setmask()
 void FixPythonInvoke::end_of_step()
 {
   PyUtils::GIL lock;
+  char fmt[] = "O";
 
-  PyObject * result = PyObject_CallFunction((PyObject*)pFunc, "O", (PyObject*)lmpPtr);
+  PyObject * result = PyObject_CallFunction((PyObject*)pFunc, fmt, (PyObject*)lmpPtr);
```

Comment:
@akohlmey Just make it a cast like the others. I dislike it either way, but that's the price we pay for Python 2 support.

Reply:
thanks for reminding me. this was my intention, but over all the other changes, I must have forgotten to change this back.